### ISSUE TITLE
Course - Reflow Issues (WCAG 1.4.10) 

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/Adapt-accessibilityfixes#64]
When the user opens a pop up window at 400% they are not able to see the entire window and cannot scroll down to see missing information.

in ie11:
There is some overlapping text when the browser is resized to 400% (long header text)
